### ISSUE TITLE
Add google analytics on the project

### ISSUE THREAD

**Issue by catiatpereira** [ipfs-shipyard/pm-idm#98]
## Description
Add google analytics on the project to track the typography usage

## Acceptance Criteria

- [ ] Acceptance Criteria 1

**Comment by daviddias** [comment]
@olizilla would Countly be a better option here?

**Comment by olizilla** [comment]
Yes! I think IDM should consider using a self hosted analytics service. You could use like https://count.ly/

If you're happy to share the ipfs one we could create a token for it on the countly.ipfs.io instance.
